ext/gtest/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 2.8.8)
```

Comment:
minimum required version is 3.1 on the main CMake, I don't know if it really matters